storybook/preview.tsx

```diff
@@ -33,6 +33,8 @@ addParameters({
       method: "alphabetical",
       order: [
         CATEGORIES.introduction,
+        CATEGORIES.systems,
+        CATEGORIES.tailwind,
```

Comment:
Hm, maybe it should be a folder in Systems: `Systems/Tailwind`?

Reply:
Moved all Tailwind stories under `Systems/Tailwind`. LayoutsAndSpacing is a sibling to Tailwind